docker/contract-verifier/Dockerfile

```diff
@@ -63,10 +63,10 @@ RUN mkdir -p /etc/zksolc-bin/v1.3.9 \
     && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.9 \
     && mv zksolc-linux-amd64-musl-v1.3.9 /etc/zksolc-bin/v1.3.9/zksolc \
     && chmod +x /etc/zksolc-bin/v1.3.9/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.10 \
```

Comment:
Please, do not remove old zksolc version from Dockerfile

Reply:
 ah makes sense, you need all versions to be present, i'll fix this